src/mgr/MgrStandby.cc

```diff
@@ -160,8 +161,32 @@ void MgrStandby::send_beacon()
   auto addr = available ? active_mgr->get_server_addr() : entity_addr_t();
   dout(10) << "sending beacon as gid " << monc.get_global_id()
 	   << " modules " << modules << dendl;
-
+  
+  const char *host_ip = NULL;
   map<string,string> metadata;
+  auto addr_standby = monc.get_my_addr();
+
+  switch(addr_standby.get_family()) {
+  case AF_INET:
+    char addr_buf_inet[INET_ADDRSTRLEN];
+    host_ip = inet_ntop(AF_INET, &addr_standby.in4_addr().sin_addr, addr_buf_inet,
+                        INET_ADDRSTRLEN);
+    break;
+  case AF_INET6:
+    char addr_buf_inet6[INET6_ADDRSTRLEN];
+    host_ip = inet_ntop(AF_INET6, &addr_standby.in6_addr().sin6_addr, addr_buf_inet6,
+                        INET6_ADDRSTRLEN);
+    break;
+  default:
+    break;
+  };
+  if (NULL == host_ip) {
+    metadata["addr"] = "0.0.0.0";
```

Comment:
If the addr isn't AF_INT or AF_INET6, then it's probably not IP at all (an RDMA messenger or something like that), so I'd just return an empty string rather than something IP-like.